webextension/popup.html

```diff
@@ -140,6 +140,27 @@
           color: #777777;
           text-decoration: none;
       }
+      .hiddenSpellError {
+          border-left-color: rgba(255,0,0, 0.2);
+          border-left-style: solid;
+          cursor: pointer;
+      }
+
```

Comment:
but it seems this blank line should be removed so that the `.hiddenXError` are grouped together.